eciYali/scripts/changeMedia_batch.m

```diff
@@ -0,0 +1,115 @@
+function [model,pos] = changeMedia_batch(model,c_source,media,flux)
+% changeMedia_batch
+%
+% function that modifies the ecModel and makes it suitable for batch growth
+% simulations on different carbon sources.
+%
+% model:  An enzyme constrained model
+% meadia: Media type ('YEP' for complex, 'MAA' minimal with Aminoacids,
+%                          'Min' for minimal media)
+% flux:   (Optional) A cell array with measured uptake fluxes in mmol/gDwh
+%
+% model: a constrained ecModel
+%
+% usage: [model,pos] = changeMedia_batch(model,c_source,media,flux)
+%
+% Ivan Domenzain        2019-05-23
+
+% Give the carbon source (c_source) input variable with the following
+% format: c_source  = 'D-glucose exchange (reversible)'
+
+if nargin<3
+    media = 'Min';
+end
+%first block any uptake
+[rxnIDs,exchange]  = getExchangeRxns(model);
+%Exclude protein pool from exchange reactions list
+protIndex = find(contains(model.rxnNames,'prot_'));
+exchange  = setdiff(exchange,protIndex);
+%First allow any exchange (uptakes and secretions)
+model.ub(exchange) = Inf;
```

Comment:
after discussion in https://github.com/SysBioChalmers/GECKO/issues/102 I think high bounds should be defined as `+1000` instead of `Inf` (same goes for several lines below).